### ISSUE TITLE
rollup does not watch files referenced from .vue SFC files

### ISSUE THREAD

**Issue by neryams** [vuejs/rollup-plugin-vue#396]
### Version
5.0.0

### Reproduction link
[https://codesandbox.io/s/rollup-plugin-vue-sfc-references-watch-nr8dr](https://codesandbox.io/s/rollup-plugin-vue-sfc-references-watch-nr8dr)

### Steps to reproduce
Create a new rollup project using vue and rollup-plugin-vue. Create a .vue file with the following format and bootstrap the app using Vue:
```
<template>
</template>

<script lang="ts" src="./app.ts"></script>
<style  lang="scss" src="./app.scss" scoped></style>
```

Create the two files app.ts and app.scss in the same folder as the .vue file. Run the project rollup with --watch. 

### What is expected?
When running with --watch, rollup will rebuild after app.ts and app.scss are modified.

### What is actually happening?
When modifying the .vue file, rollup build will recompile. But when modifying the .ts or .scss files, rollup will not recompile.

**Comment by neryams** [comment]
https://github.com/vuejs/rollup-plugin-vue/pull/385

**Comment by znck** [comment]
fixed in #385
